content/scalability/docs/control-plane.md

```diff
@@ -240,4 +244,50 @@ spec:
 
 - This FlowSchema captures all list event calls made by service accounts in the default namespace.
 - The matching precedence 8000 is lower than the value of 9000 used by the existing service-accounts FlowSchema so these list event calls will match list-events-default-service-accounts rather than service-accounts.
-- We're using the catch-all PriorityLevelConfiguration to isolate these requests. This bucket only allows 13 inflight requests to be used by these long-running list event calls. Pods will start to receive 429s as soon they try to issue more than 13 of these requests concurrently. 
+- We're using the catch-all PriorityLevelConfiguration to isolate these requests. This bucket only allows 13 inflight requests to be used by these long-running list event calls. Pods will start to receive 429s as soon they try to issue more than 13 of these requests concurrently.
+
+## Use Shared Informers for Custom Controllers
```

Comment:
Should we add another section here to capture with a heading something like "Monitor resources in API server" and these two could be sub-headings to this?